torch_patches/.torch_pin

```diff
@@ -0,0 +1 @@
+#73164
```

Comment:
This means this PR works only with the change in https://github.com/pytorch/pytorch/pull/73164. Then would you check in this file when you merge this PR, or would you wait until https://github.com/pytorch/pytorch/pull/73164 merges first and then discard change in this torch_patches/.torch_pin?

Reply:
I believe the Later. You can check out this guide for more details. https://github.com/pytorch/xla/blob/master/.circleci/README.md.

Reply:
Thanks!